NEWS.md

```diff
@@ -62,6 +62,8 @@ Library improvements
 
   * New `accumulate` and `accumulate!` functions, which generalize `cumsum` and `cumprod`. Also known as a [scan](https://en.wikipedia.org/wiki/Prefix_sum) operation ([#18931]).
 
+  * New `titlecase` function, which capitalizes the first character of each word within a string ([#19469]).
```

Comment:
No, we are holding off on running that... we will update a bunch of links at once in a separate PR.  Updating the NEWS links in individual PRs creates too many merge conflicts.